tools/wine.in

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Wrapper is used because `CMAKE_CROSSCOMPILING_EMULATOR` accepts lists starting with CMake 3.15. Before, a command with arguments would be interpreted as just the name of a command.